packages/perseus/src/widgets/matrix/score-matrix.test.ts

```diff
@@ -1,13 +1,66 @@
 import {mockStrings} from "../../strings";
 
 import scoreMatrix from "./score-matrix";
+import * as MatrixValidator from "./validate-matrix";
```

Comment:
Yes. When you want to mock a function on an imported module, you need to use the "star import" syntax so that you can spy on a single function from that module.